libs/controller/controller.py

```diff
@@ -322,13 +324,13 @@ def delete_current_profile(self, equipment_id=None):
         self.push_profile_old_method(equipment_id=equipment_id)
         self.delete_profile(profile_id=delete_ids)
 
-    # This will delete all the profiles on an controller_tests instance, except the default profiles
+    # This will delete all the profiles on an controller instance, except the default profiles
     def cleanup_profiles(self):
         try:
             self.get_default_profiles()
             pagination_context = """{
                             "model_type": "PaginationContext",
-                            "maxItemsPerPage": 5000
+                            "maxItemsPerPage": 10000
```

Comment:
Why did we increase this?

Reply:
Shouldn't we delete the profiles that the test cares about? And not all profiles? IN case there is content on the same resource.

Reply:
yes so this method is not used as a part of our sanity, but to make sure that our controller instance is free from all profiles excluding the default ones. as a part of general cleanup. I added that so that it saves time to clean one by one if we want for to do manually